crates/autopilot/src/arguments.rs

```diff
@@ -167,6 +167,13 @@ pub struct Arguments {
     /// limit orders.
     #[clap(long, env, use_value_delimiter = true)]
     pub quoting_strategies: Vec<QuotingStrategy>,
+
+    /// Fee scaling factor for objective value. This controls the constant
+    /// factor by which order fees are multiplied with. Setting this to a value
+    /// greater than 1.0 makes settlements with negative objective values less
+    /// likely, promoting more aggressive merging of single order settlements.
+    #[clap(long, env, default_value = "1", value_parser = shared::arguments::parse_unbounded_factor)]
+    pub fee_objective_scaling_factor: f64,
```

Comment:
I think this change was also done on `main` recently. Specifically in this PR: https://github.com/cowprotocol/services/pull/1199

Maybe the feature branch should be rebased instead of including it here in this PR?

Reply:
Resolved conflicts.